app_sensor_module/src/main.c

```diff
@@ -64,8 +60,10 @@ static int init(void) {
     return 0;
 }
 
-int main() {
-    if (init()) {
+int main(   )
+{
+    if (init())
+    {
```

Comment:
Bro fell asleep on his keyboard.

Reply:
Lol was intentional to try and get the CI to fix those. Gonna close this PR for now. Was a WIP that isnt working yet